include/module_hello.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021 Legrand North America, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ *  @brief Module Hello API
+ */
+#ifndef EXAMPLE_APPLICATION_INCLUDE_MODULE_HELLO_H_
+#define EXAMPLE_APPLICATION_INCLUDE_MODULE_HELLO_H_
+
+#include <stdint.h>
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup isr_apis Interrupt Service Routine APIs
+ * @ingroup kernel_apis
```

Comment:
What groups should be used here, considering this is a test case for the Zephyr document generation?